user/reference/bionic.md

```diff
@@ -3,6 +3,8 @@ title: The Ubuntu 18.04 Build Environment
 layout: en
 ---
 
+> Bionic LTS Standard is EOL by Canonical, try updating to a newer image. It will be in use in Travis CI for some time more, but it's time to consider migration of your builds to a newer build environment image.
```

Comment:
Canonical has ended support for Bionic LTS Standard. It’s time to update to a newer image. Although Travis CI will continue using it for a while, consider migrating your builds to a more current environment.